wagtail/users/views/users.py

```diff
@@ -345,7 +345,7 @@ def get_page_subtitle(self):
 class UserViewSet(ModelViewSet):
     icon = "user"
     model = User
-    ordering = "name"
+    ordering = User.USERNAME_FIELD
```

Comment:
Thanks @d4x1  I think this fix makes sense, are you able to include a unit test for this also?

All the pieces should be in the codebase already for you to create this test.

* https://github.com/wagtail/wagtail/blob/main/wagtail/test/customuser/models.py#L59 `CustomUser` model with `USERNAME_FIELD = "username"` already set.
* https://github.com/wagtail/wagtail/blob/main/wagtail/test/customuser/viewsets.py `CustomUser` viewset that extends `UserViewSet`.
* https://github.com/wagtail/wagtail/blob/main/wagtail/users/tests/test_admin_views.py#L2740 -> an existing set of unit tests using a `CustomUserViewSet` - I assume you could just add another test scenario to this test class, under `test_get_viewset_cls_custom` or in that area.